lib/min_heap.rb

```diff
@@ -39,33 +43,64 @@ def to_s
     end
 
     output += @store.last.value + "]"
-      
+
     return output
   end
 
   # This method returns true if the heap is empty
-  # Time complexity: ?
-  # Space complexity: ?
+  # Time complexity: O(n), where n is the number of nodes in the heap
```

Comment:
Are you adding n nodes to the list, or counting them?  No, you're just using `.length` which is an array instance variable.  So this is actually O(1)